plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/Core.melk

```diff
@@ -168,6 +168,13 @@ programmatic option bendPoints: KVectorChain {
     targets edges
     legacyIds de.cau.cs.kieler.bendPoints
 }
+advanced option contentAlignment: EnumSet<ContentAlignment> {
+    label "Content Alignment"
+    description "Specifies how the content of compound nodes is to be aligned, e.g. top-left."
+    default = EnumSet.of(ContentAlignment.H_CENTER, ContentAlignment.V_TOP)
```

Comment:
I don't think this should be the global default since it change's ELK Layered's behaviour. I'd prefer the Box and RectPacking algorithms to specify custom defaults.

Reply:
Ok.